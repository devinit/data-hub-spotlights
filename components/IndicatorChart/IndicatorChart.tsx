import React, { FunctionComponent } from 'react';

const IndicatorChart: FunctionComponent = () => {
  return (
    <div className="l-2up-3up__col">
      <div className="spotlight__stat">
        <div className="spotlight__interactive"></div>
      </div>
    </div>
  );
};

export { IndicatorChart };
