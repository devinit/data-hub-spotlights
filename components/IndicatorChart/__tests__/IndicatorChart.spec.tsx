/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { IndicatorChart } from '../IndicatorChart';

describe('IndicatorChart', () => {
  test('renders default state correctly', () => {
    const renderer = TestRenderer.create(<IndicatorChart />).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
