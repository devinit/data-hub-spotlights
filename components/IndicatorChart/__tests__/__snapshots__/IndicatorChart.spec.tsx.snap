// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndicatorChart renders default state correctly 1`] = `
<div
  className="l-2up-3up__col"
>
  <div
    className="spotlight__stat"
  >
    <div
      className="spotlight__interactive"
    />
  </div>
</div>
`;
